app/views/dashboard/index.html.erb

```diff
@@ -222,8 +222,10 @@
                     <%= total_received_from_product_drives %>
                   </span> items
                   received <%= @selected_date_range_label %></h3>
-                <h4 class="text-center"><%= dollar_presentation(total_received_money_donations) %>
-                  raised  <%= @selected_date_range_label %></h4>
+                <h4 class="text-center">
+                  <span class="total_money_raised">
+                    <%= dollar_presentation(total_received_money_donations_from_product_drives) %>
```

Comment:
This is out of scope but I wanted to mention that I see we have the gem for handling monetary values and we have a homemade `dollar_presentation` helper method. Seems like to me it is a good idea to break away from home-built solutions and relies on the gem in the future.

A refactor job in the future :)

Reply:
Yeah,  I'll throw it into the huge pile.